### ISSUE TITLE
Integrate MDTranslator into Datagov Harvesting Logic

### ISSUE THREAD

**Issue by btylerburton** [GSA/data.gov#4565]
## User Story

In order to successfully transform datasets from one schema to another, datagovteam would like to use the MDTranslator library, via a Rails application, to do so.

Depends on: 
- https://github.com/GSA/data.gov/issues/4467

## Acceptance Criteria

- [ ] GIVEN a harvest source with an ISO19115-2 schema, stored in a WAF file format
  WHEN that source has been loaded into the harvest runner, which does not include the validate and load steps
  THEN the harvest runner will utilize the MDTranslator Rails application to transform the source into valid [DCATUS](https://resources.data.gov/resources/dcat-us/) format. 
  
## Background
At present, this additional work to support deferred operations is not necessarily a given, and could be a future enhancement to unblock pipeline workers while the transform is processing.

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))

None. All work will reside within the Cloud.gov boundary and no external routes should be necessary.

## Sketch

- [x] Integrate the MDTranslator rails application into a local Docker Compose container network
- [x] Test that local integration fully before proceeding to Cloud.gov integration
- [ ] Implement the same locally tested solution in Cloud.gov
  - [ ] Note that this will involve the creation of new internal routes and network mappings, and it is encouraged to keep these considerations in mind while developing the local service
- [ ] Configure a WAF Harvest source using ISO19115-2 standard metadata
- [ ] Extract that WAF using datagov-harvesting-logic WAF extract task
- [ ] Transform datasets into DCATUS using the MDTranslator application
- [ ] Check for errors

**Comment by rshewitt** [comment]
[#4940](https://github.com/GSA/data.gov/issues/4940) reminded me to configure the rails app for production ( e.g. `rails server -e production` ). i'll look into this later.

**Comment by rshewitt** [comment]
this could include publishing an image of mdtranslator and adding it to the docker compose file

**Comment by rshewitt** [comment]
spoke with james and tyler and we've decided to update the values for schema_type & source_type. schema_type can be `"iso19115_1", "iso19115_2", "csdgm", "dcatus"` and source_type can be `"document", "waf"`.

**Comment by Bagesary** [comment]
MOved to Sprint backlog to focus on non-harvester stories
